### DISCUSSION TITLE
Question: for the cluster_pathways why do some nodes of multiple colors and some are just solid one color?

### DISCUSSION THREAD

**Discussion by ahdee** [egeulgen/pathfindR#18] [Q&A] [answered]
Sorry a bit confused since there were not legend.  Do the nodes with split color indicate that the pathway can belong to another cluster as well? thanks!

**Comment by egeulgen** [comment, marked as answer]
yes. if you're using fuzzy clustering, nodes (representing pathways) with multiple colors indicate the different clusters this pathway belongs to.

**Comment by ahdee** [comment]
thanks! two more questions? 
1. what does  the size represent? number of genes for that pathway or p.value?
2. do the distance of the nodes indicate how related they are ?

**Comment by egeulgen** [comment]
1. Size of node corresponds to -log(lowest_p), i.e. the bigger, the more significant
2. Not the distance but the thickness of the edges between nodes correspond to the kappa statistic between the two terms (i.e. how related they are)

You may read more details in `?cluster_graph_vis`
